operations/identitydocuments.md

```diff
@@ -0,0 +1,331 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Identity documents
+
+## Get all identity documents
+
+Returns all identity documents. This operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/identityDocuments/getAll`
+
+```javascript
+{
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "Client": "Sample Client 1.0.0",
+  "IdentityDocumentIds": [
+    "e8a72a69-c20b-4278-b699-ab0400a32ecc",
+    "24a3f051-49ed-411b-9384-78187f9daae6"
+  ],
+  "CustomerIds": [
+    "fadd5bb6-b428-45d5-94f8-fd0d89fece6d",
+    "bccdafd1-3e44-439d-861f-341526b597a9"
+  ],
+  "ChainIds": [
+    "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+    "5fcd1933-22f2-40b9-84da-7db04cbecec2"
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ChainIds` | array of string | optional, max 1000 items | Unique identifiers of the Chains. If not specified, the operation returns data for all chains within scope of the Access Token. |
```

Comment:
My suggestion here is just for consistency with the way the other fields are described.
```suggestion
| `ChainIds` | array of string | optional, max 1000 items | Unique identifiers of `Chain`. If not specified, the operation returns data for all chains within scope of the Access Token. |
```